docs/third_party.rst

```diff
@@ -56,6 +56,9 @@ aiohttp extensions
 
 - `aiohttp-jinja2 <https://github.com/aio-libs/aiohttp-jinja2>`_ Jinja2
   template renderer for aiohttp.web.
+  
+- `aiohttp-grapqhql <https://github.com/graphql-python/aiohttp-graphql>`_
```

Comment:
Typo in project name :)

Reply:
Nice found, @socketpair !

Reply:
It nicely mirrors how often I mistype `graphql` in vim. Thanks for finding!